packages/clarity-js/src/interaction/scroll.ts

```diff
@@ -1,14 +1,17 @@
-import { Event } from "@clarity-types/data";
+import { Dimension, Event } from "@clarity-types/data";
 import { ScrollState, Setting } from "@clarity-types/interaction";
 import { bind } from "@src/core/event";
 import { schedule } from "@src/core/task";
 import { time } from "@src/core/time";
 import { clearTimeout, setTimeout } from "@src/core/timeout";
-import { iframe, get } from "@src/layout/dom";
-import { target } from "@src/layout/target";
+import { iframe } from "@src/layout/dom";
+import { target, metadata } from "@src/layout/target";
 import encode from "./encode";
+import * as dimension from "@src/data/dimension";
 
 export let state: ScrollState[] = [];
+let initTopNode: Node;
```

Comment:
>initTopNode [](http://example.com/codeflow?start=4&length=11)

nit: I'd really prefer initialTopNode to initTopNode, it sounds like you are going to initialize something

Reply:
you can also drop node and just do initialTop and Bottom